src/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { createRoot } from "react-dom/client";
+import { Provider } from "react-redux";
+
+import App from "./components/App";
+import store from "./app/store";
+import "./css/index.css";
+
+const rootDiv = document.getElementById("root");
+if (!rootDiv) {
+  throw new Error("Root container missing in index.html");
+}
+
+const root = createRoot(rootDiv);
+root.render(
```

Comment:
In React 18, the render method has changed slightly. It no longer uses ReactDOM, but a createRoot object. See https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis